requirements/doc.txt

```diff
@@ -62,7 +62,7 @@ idna==3.3
     # via requests
 imagesize==1.3.0
     # via sphinx
-importlib-metadata==4.11.4
+importlib-metadata==4.12.0
```

Comment:
No breaking change mentioned in the new version https://github.com/python/importlib_metadata/blob/main/CHANGES.rst#v4120